networking/cidr-range-definitions.adoc

```diff
@@ -22,14 +22,14 @@ endif::[]
 
 [IMPORTANT]
 ====
-OVN-Kubernetes, the default network provider in {product-title} 4.11 and later, uses the `100.64.0.0/16` IP address range internally. If your cluster uses OVN-Kubernetes, do not include the `100.64.0.0/16` IP address range in any other CIDR definitions in your cluster.
+OVN-Kubernetes, the default network provider in {product-title} 4.11 and later, uses the following IP address ranges internally: `100.64.0.0/16`, `169.254.169.0/29`, `100.88.0.0/16`, `fd98::/64`, and `fd97::/64`. If your cluster uses OVN-Kubernetes, do not include any of the previous ranges in any other CIDR definitions in your cluster.
```

Comment:
1. `100.88.0.0/16` and `fd97::/64` only from 4.14 version.  so if this will backport to 4.12. please remove them
2.  sounds like missed ipv6 MasqueradeSubnet `fd69::/125`

Reply:
It sounds like this should only apply to 4.14+ then. Is that more accurate?

Reply:
only 100.88.0.0/16 and fd97::/64 for 4.14+
for others CIDR, should be 4.12+

Reply:
So like this: 
4.14-4.16 = only `100.88.0.0/16` and `fd97::/64`
4.12-4.13 = `100.64.0.0/16`, `169.254.169.0/29`, `100.88.0.0/16`, `fd98::/64`, `fd69::/125`, and `fd97::/64`

@zhaozhanqi

Reply:
4.14-4.16 = 100.88.0.0/16 and fd97::/64 , 100.64.0.0/16, 169.254.169.0/29, fd98::/64, fd69::/125
4.12-4.13 = 100.64.0.0/16, 169.254.169.0/29,  fd98::/64, fd69::/125